<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PilveenSiirto</class>
 <widget class="QDialog" name="PilveenSiirto">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kirjanpidon kopiointi pilveen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/pic.qrc">
    <normaloff>:/pic/cloud.png</normaloff>:/pic/cloud.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nimiLabel">
       <property name="text">
        <string>Nimi</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>640</width>
         <height>90</height>
        </rect>
       </property>
       <property name="text">
        <string>Tämä toiminto kopioi nykyisen kirjanpitosi Kitsaan pilveen. Tämän jälkeen kirjanpito varmuuskopioidaan automaattisesti, voit käyttää laskujen kierrätystä ja kirjanpitoa voidaan käsitellä monelta tietokoneelta.

Nykyinen kirjanpitotiedosto jää edelleen tietokoneellesi.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="infoLabel">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>105</y>
         <width>621</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="sijaintiGroup">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>129</y>
         <width>621</width>
         <height>165</height>
        </rect>
       </property>
       <property name="title">
        <string>Kirjanpidon sijainti</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="omaRadio">
          <property name="text">
           <string>Henkilökohtainen kirjanpito</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/mies.png</normaloff>:/pic/mies.png</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="toimistoRadio">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Tilitoimiston kirjahylly</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/kansiot.png</normaloff>:/pic/kansiot.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="toimistoLabel">
            <property name="text">
             <string>Tilitoimisto</string>
            </property>
            <property name="buddy">
             <cstring>toimistoCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="toimistoCombo">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hyllyLabel">
            <property name="text">
             <string>Kirjahylly</string>
            </property>
            <property name="buddy">
             <cstring>hyllyCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="hyllyCombo">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>58</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1" columnminimumwidth="40,0">
         <item row="0" column="0">
          <widget class="QLabel" name="rasti1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="vaihe1">
           <property name="text">
            <string>Kopioidaan tilikarttaa ja asetuksia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="rasti2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="vaihe2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Kopioidaan tositteita</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="vaihe3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Kopioidaan liitteitä</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>207</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="valmisLabel">
         <property name="text">
          <string>Kirjanpitosi on kopioitu Kitsaan pilveen.

Myös paikallinen kirjanpitotiedosto on edelleen jäljellä. Huomaathan, että paikalliseen kirjanpitoon tehdyt muokkaukset eivät muuta pilvessä olevaa kirjanpitoa. Pilvessä olevaa kirjanpitoa ei voi myöskään voi muuntaa takaisin paikalliseksi kirjanpitotiedostoksi.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valmisInfo">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>222</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="isotTiedostot">
         <property name="text">
          <string>Kirjanpitoa ei voi siirtää pilveen. 

Kirjanpidossa on yli 10 megatavun kokoisia liitteitä, joita ei voi 
tallentaa Kitsaan pilveen.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="isoLista"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Help|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toimistoRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>toimistoCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toimistoRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>hyllyCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
